source/strings.tex

```diff
@@ -1433,22 +1401,19 @@
 
 \begin{itemdescr}
 \pnum
-\effects Constructs an object of class \tcode{basic_string} as indicated in
-Table~\ref{tab:strings.ctr.6}. The stored allocator is constructed from
-\tcode{alloc}. In the second form, \tcode{str} is left in a valid state with an
+\effects Constructs an object of class \tcode{basic_string}.
+The stored allocator is constructed from \tcode{alloc}.
+In the second form, \tcode{str} is left in a valid state with an
```

Comment:
For the corresponding constructors without an allocator parameter, this sentence is a postcondition rather than an effect. We should be consistent.

Reply:
Done.